src/components/Form.tsx

```diff
@@ -21,31 +22,18 @@ export default function Form() {
   const [isSend, setIsSend] = useState(false);
 
   // TODO: convert this handler to match TS, fix state conditions
-  function handleInputChange(e: FormEvent) {
-    setName({ [e.target.name]: e.target.value });
+  function handleSelectChange(
+    e: React.ChangeEvent<HTMLSelectElement>,
+    fc: (a: string) => void
```

Comment:
callback? Why?

Reply:
I wanted to make this function generic for all selects (right now we've got 1 select, but in future this funcion could handle all of them)

Reply:
No way :)